docs/translations-needed.md

```diff
@@ -252,16 +252,8 @@ If you add or update a translation run `grunt check-translations` to update the
 |                         | , opens subtitles settings dialog                                                   |
 |                         | , opens descriptions settings dialog                                                |
 |                         | , selected                                                                          |
-| ru.json (missing 14)    | Close Modal Dialog                                                                  |
-|                         | Descriptions                                                                        |
-|                         | descriptions off                                                                    |
-|                         | Audio Track                                                                         |
+| ru.json (missing 6 )    | Close Modal Dialog                                                                  |
 |                         | The media is encrypted and we do not have the keys to decrypt it.                   |
```

Comment:
You can see an example of this by running the following snippet:
```js
player.error({code: 5});
```
It'll open an error dialog with this text in it.
To close the dialog, you call `error()` with `null`:
```js
player.error(null);
```.